### PR TITLE
request to add israel

### PR DESCRIPTION
**Pull request by nomq1**
please see issue #769 ,
as per Hungary example , I added icon and description .

### CODE REVIEW

**Comment by nomq1** [comment]
I accept the [Contributor License Agreement](https://github.com/grote/Transportr/blob/master/CLA.md)

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -395,6 +395,11 @@ and always knows where you are to not miss where to get off the bus.
 	<string name="np_desc_hungary">Budapest, Miskolc</string>
 	<string name="np_desc_hungary_networks" translatable="false">BKK, MKV</string>
 
+	<string name="np_name_israel">Israel</string>
+	<string name="np_desc_israel">Jerusalem</string>
```
Comment (pt2121) [inline app/src/main/res/values/strings.xml:399]: Out of curiosity, does this provider only serve in Jerusalem?

File: app/src/main/java/de/grobox/transportr/networks/TransportNetworks.kt [hunk 2]
```diff
@@ -757,6 +757,18 @@ private val networks = arrayOf(
                         factory = { DubProvider() }
                     )
                 )
+            ),Country(
```
Comment (pt2121) [inline app/src/main/java/de/grobox/transportr/networks/TransportNetworks.kt:760]: nit: formatting

```suggestion
            ),
            Country(
```

**Review by pt2121 — COMMENTED** [review]
Sorry for the delay and thanks for the contribution. If you can include some screenshots in the PR comment/description, it'd be awesome.

**Comment by pt2121** [comment]
Again, thanks for the contribution. I'm closing this as there is no activity. Feel free to reopen if you're still interested in working on it.

**Comment by TalOrenshtein** [comment]
[looks like it cover all of Israel ](https://github.com/schildbach/public-transport-enabler/issues/201)

**Comment by thePrivacyFanatic** [comment]
just so i understand, is this waiting on  public-transport-enabler or complete and just not merged?

**Pull request closed without merging**